docker/generic/Dockerfile.base

```diff
@@ -79,15 +83,70 @@ RUN echo "source /opt/ros/$ROS_DISTRO/setup.bash" >> /etc/profile.d/ros.sh && \
 #
 # Scan Autoware packages to install dependencies
 #
+# Ignore ament dependencies for now
 COPY ./ /tmp/Autoware
 RUN apt-get update && \
         apt-get install -y ros-$ROS_DISTRO-desktop-full && \
-        rosdep install -y --from-paths /tmp/Autoware/ros/src --ignore-src && \
+        rosdep install -y --from-paths /tmp/Autoware/ros/src --ignore-src --skip-keys="ament_lint_common ament_lint_auto ament_cmake_gtest ament_cmake ament_cmake_lint_cmake" && \
         rm -rf /var/lib/apt/lists/* && \
         rm -rf /tmp/Autoware  # Removed for clean environment
 
 RUN su -c "rosdep update" autoware
 
+#
+# Enable ament_cmake
+#
+
```

Comment:
Add ROS2's GPG key, see: https://github.com/osrf/docker_images/blob/4f09cf3ebe9300879f9a27eca2c60632ff980431/ros/ardent/ubuntu/xenial/ros-core/Dockerfile

Reply:
We use FROM ros:$ROS_DISTRO here, so ROS GPG key is already imported:
<pre>
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 421C365BD9FF1F717815A3895523BAEEB01FA116
Executing: /tmp/tmp.3Qb23No1Wz/gpg.1.sh --keyserver
hkp://keyserver.ubuntu.com:80
--recv-keys
421C365BD9FF1F717815A3895523BAEEB01FA116
gpg: requesting key B01FA116 from hkp server keyserver.ubuntu.com
gpg: key B01FA116: "ROS Builder <rosbuild@ros.org>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
</pre>